lib/livebook/storage/ets.ex

```diff
@@ -98,13 +110,90 @@ defmodule Livebook.Storage.Ets do
 
     :ets.insert(table, attributes)
 
-    {:reply, :ok, state}
+    {:reply, :ok, schedule_persist(state)}
```

Comment:
Since we control the functions that mutate the storage, it probably makes sense to schedule it in those cases (unless scheduled), so that we don't encrypt and write unnecessarily.